### ISSUE TITLE
Importing files from a symlinked folder not supported?

### ISSUE THREAD

**Issue by adrianmcli** [zeit/next.js#3168]
I get a "module not found" error whenever I try to import a file from a symlinked folder:

![screen shot 2017-10-24 at 8 29 57 pm](https://user-images.githubusercontent.com/943555/31979188-30f86246-b8fa-11e7-880f-47d34e3fa7d2.png)

- [x] I have searched the [issues](https://github.com/zeit/next.js/issues) of this repository and believe that this is not a duplicate.

## Expected Behavior
Regardless of whether the folder is a symlink or not, the importing should work as if it was a normal folder.

## Current Behavior
It cannot find the javascript file inside the symlink folder.

## Steps to Reproduce (for bugs)

I've made a small example here just to demonstrate the issue: https://github.com/adrianmcli/nextjs-symlink-bug

In order to reproduce this bug, clone the repo and run `yarn` or `npm install`, and then run `yarn dev` to run the app.

1. Notice that `lib/symlink-folder` is a symlinked folder.
2. Go into `pages/index.js` and notice that it currently refers to `lib/normal-folder`.

   ```js
   import someObj from '../lib/normal-folder/someText'
   ```

3. Change that import statement to refer to `lib/symlink-folder` instead.

   ```js
   import someObj from '../lib/symlink-folder/someText'
   ```

4. Notice that the app gives you a `module not found` error.

## Context
I am trying to create a boilerplate project with Truffle and Next.js for some Dapp development. You can see my work-in-progress here: https://github.com/adrianmcli/truffle-next

**Comment by adrianmcli** [comment]
While a little disappointed that barely anyone even looked at this issue, I decided to dig a bit further after letting the problem sit for a few days.

After some googling, I found out that it may be an issue related to Webpack:
https://github.com/webpack/webpack/issues/1643

[This comment](https://github.com/webpack/webpack/issues/1643#issuecomment-317436595) in that issue said that I needed to set `resolve.symlinks: false` in my webpack configuration. So that's exactly what I did inside `next.config.js`:

```js
module.exports = {
  webpack: (config, { buildId, dev }) => {
    config.resolve.symlinks = false
    return config
  }
}
```

And now it works, yay!
